src/sentry/grouping/enhancer/__init__.py

```diff
@@ -49,7 +49,7 @@
 enhancements_grammar = Grammar(
     r"""
 
-enhancements = line+
+enhancements = line*
```

Comment:
The new version of parsimonious correctly handles the quantifiers `*` and `+` and before `+` was somehow interpreted as `*` so we changed the grammar to account for it.